packages/plugin-phone/test/integration/spec/media-state-controls.js

```diff
@@ -460,10 +466,13 @@ describe(`plugin-phone`, function() {
                   assert.equal(mccoyCall.remote.status.audioStatus.toLowerCase(), boolToStatus(true, true));
                   assert.equal(mccoyCall.remote.status.videoStatus.toLowerCase(), boolToStatus(true, false));
                   return Promise.all([
-                    call.toggleSendingAudio(),
```

Comment:
I originally wrote this incorrectly. The goal was to toggle a media type, not a direction